tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -363,24 +363,17 @@ uint32_t TypeTranslator::getSpirvBitwidth(QualType type) {
     case BuiltinType::LongLong:
     case BuiltinType::ULongLong:
       return 64;
-    // min16int (short), and min12int are treated as 16-bit Int if
+    // min16int (short), ushort, and min12int are treated as 16-bit integers if
     // '-enable-16bit-types' option is enabled. They are treated as 32-bit
-    // Int otherwise.
+    // integers otherwise.
     case BuiltinType::Short:
+    case BuiltinType::UShort:
```

Comment:
Oh, sorry. Also `Half` and `Min10Float`?

Reply:
Oh sorry i don't know why i missed that!